odl/set/sets.py

```diff
@@ -310,6 +310,25 @@ def field(self):
         """
         return self
 
+    def contains_set(self, other):
+        raise NotImplementedError('field {!r} does not have method '
+                                  '`contains_set`'.format(self))
+
+    def __add__(self, other):
+        if self.contains_set(other):
+            return self
+        elif other.contains_set(self):
+            return other
+        else:
+            raise ValueError('Fields {!r} and {!r} do not include '
+                             'each other'.format(self, other))
```

Comment:
I'm skeptical towards this. It doesn't fulfill the rules for set addition, and just as a shortcut for something used in `OperatorSum`, it shouldn't introduce an API change for `Field`.

Reply:
I am adding fields, not sets and so I thought I went with the _Minkowski addition_ `A + B = {a + b | a in A, b in B}` (which at least for the implemented numbers is trivial). I actually haven't thought about other sets.

What would be your suggestion to approach this? At first I did something like `if isinstance(a.range, ComplexNumbers) or isinstance(b.range, ComplexNumbers): return ComplexNumbers(); elif ...` but that was rather unpleasant to look at...

Reply:
I see... The spaces are homomorph but not isomorph. You were right all along...

Reply:
:rofl: You went on quite a journey there, my mention of finite fields was more like an example.. In some sense you proved my minimal point that it's not trivial to implement set addition and keep it in line with mathematics for arbitrary fields.

But it doesn't seem to be necessary here anyway.